package.py

```
@@ -682,98 +688,93 @@ def main():
+    # Always build to ensure latest version
+    Colors.print("Building Neutron and Box executables...", Colors.YELLOW)
+
+    # Preflight checks on Windows to catch common vcpkg/tooling problems before we start
+    if os_type == 'windows' and not args.skip_vcpkg:
+        ok = preflight_check_windows()
+        if not ok:
+            Colors.print("Preflight checks failed. Either install the listed prerequisites or re-run with --skip-vcpkg to proceed (may still fail).", Colors.RED)
+            sys.exit(1)
+
+    # Build both executables
+    neutron_success = build_neutron(os_type, arch_type, build_dir, args.skip_vcpkg, args.debug)
```

Ruff (0.14.10)

Local variable `neutron_success` is assigned to but never used

Remove assignment to unused variable `neutron_success`

(F841)